python/pyhail/dataset.py

```diff
@@ -313,12 +497,62 @@ def annotate_samples_vds(self, right, root=None, code=None):
     def annotate_variants_bed(self, input, root, all=False):
         """Annotate variants with a .bed file.
 
+        **Examples**
+
+        First consider a standard BED file::
+
+          $ cat file1.bed
+          track name="BedTest"
+          20    1          14000000
+          20    17000000   18000000
+
+        In order to annotate with this file, the command should appear as::
+
+        >>> vds1 = (hc.read('example.vds')
+        >>>  .annotate_variants_bed('file1.bed', 'va.cnvRegion'))
+
+        This file format produces a Boolean annotation::
+
+          Variant annotations:
+          va: va.<identifier>
```

Comment:
Let's make this what the output of print_schema() would look like. I don't think we need the `<lots of other stuff here>`.